### PR TITLE
IDEM-315

### PR DESCRIPTION
**Pull request by JasminFarhi**
Update DS logo this one should be a little nicer
Screen Shot 2022-07-29 at 12 32 11 pm

#315

### CODE REVIEW

File: data/sites.json [hunk 1]
```diff
@@ -5,7 +5,7 @@
     "tagline": "Bitcoin & Crypto made simple",
     "description": "A uniquely easy way to buy, sell & store crypto in Australia. Enjoy safe & effortless trading today.",
     "website": "https://digitalsurge.com.au/",
-    "logo": "https://tradersunion.com/images/brokers/743.jpeg",
+    "logo": "https://australianfintech.com.au/wp-content/uploads/sites/7/2018/07/Digital-Surge.png",
```
Comment (bitcoinbrisbane) [inline data/sites.json:8]: Lets put this in our own repo so we cant get blocked, or our app break if it changes etc https://github.com/dltxio/idem-mobile/tree/development/data

**Review by bitcoinbrisbane — CHANGES_REQUESTED** [review]
Lets add their image to our repo

File: data/sites.json (hunk 1 above)
Comment (JasminFarhi) [reply data/sites.json:8]: Sure thing :)

**Review by bitcoinbrisbane — APPROVED** [review]

**Comment by bitcoinbrisbane** [comment]
@JasminFarhi will need a screen shot on their page too

**Comment by bitcoinbrisbane** [comment]
@JasminFarhi merged but getting 404 https://raw.githubusercontent.com/dltxio/idem-mobile/main/data/digital_surge_logo.png

**Pull request merged**